### ISSUE TITLE
Question about the output file

### ISSUE THREAD

**Issue by baoyl818** [Genotek/ClassifyCNV#1]
Dear friend,

ClassifyCNV is so great. I haved tried this excellent software, and get what I want. Here, I have a question refers to the output file. I cannot find the meaning of this column after the column 'Total score', can you add some information about the every column of the output file? Thanks very much.

VariantID
Chromosome
Start
End
Type
Classification
Total score
**1                        what's the meaning of this column?**
2A
2B......................

![image](https://user-images.githubusercontent.com/56709000/102308256-3a941580-3fa1-11eb-9eb3-a0d71a24bc40.png)

**Comment by tgurbich** [comment]
Dear @baoyl818,

Thank you for using ClassifyCNV and for your feedback. Column names 1 through 5H correspond to the numbered evidence fields of the ACMG rubrics for [copy-number loss](http://cnvcalc.clinicalgenome.org/cnvcalc/cnv-loss) and [copy-number gain](http://cnvcalc.clinicalgenome.org/cnvcalc/cnv-gain) and represent the breakdown of the total score. For column "1", I collapsed fields 1A and 1B of the ACMG rubric into one column titled "1" since fields 1A and 1B of the ACMG rubric are mutually exclusive. The rest of the columns in the ClassifyCNV results table match the numbers of the ACMG rubric fields exactly. I can see how a column titled "1" can be confusing so I renamed it to "1A-B" to match the ACMG rubric. 

ClassifyCNV outputs scores for each individual line of evidence in case there is a need to follow up on any of the calls or if it is necessary to check how the final score was achieved for a particular CNV. I expect that for many users these details might not be of interest and the total score will suffice.
